requirements.txt

```diff
@@ -0,0 +1,5 @@
+huggingface-hub==0.10.1
+transformers==4.23.0
+fastcore==1.5.27
+fastdownload==0.0.7
+git+https://github.com/huggingface/diffusers.git@main#egg=diffusers
```

Comment:
Thanks for this! Just a thought:

This requirements.txt depends on what you are starting with inside your environment.
For example, many things already come on Google Colab (scipy, matplotlib, etc.).
But if someone is setting up a fresh env (Docker, Conda, etc.), then they will need
to install other dependencies like scipy, matplotlib, Jupyter and so on.
As I was setting up a fresh Docker file this evening in order to play
with the first notebook, `stable_diffusion`, I found I needed to install other packages other than those listed here.

Reply:
I removed the entries for `fastcore` and `fastdownload` and added one for `fastai` since it brings both `fastcore` and `fastdownload`. Moreover, it brings other necessary libraries, like `torch`, `scipy`, and `matplotlib`. I don't think that installing `notebook` or `jupyterlab` is part of this `requirements.txt` file. You need Jupyter just to be able to open the `ipynb` file, and you can start from a Jupyter-provided Docker image for this. But, I'm happy to discuss it.